src/frontend/get-github-configuration.ts

```diff
@@ -65,21 +71,37 @@ async function getInstallationsWithAdmin(installations: Octokit.AppsListInstalla
 			type: installation.target_type
 		});
 
-		const authedApp = await app.auth(installation.id);
-		enhanceOctokit(authedApp);
+		if(await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_TO_COUNT_REPOS, true, jiraHost)){
+			const githubClient = new GitHubClient(installation.id, log);
+			const numberOfReposPromise = githubClient.getNumberOfReposForInstallation();
 
-		const repositories = authedApp.paginate(
-			authedApp.apps.listRepos.endpoint.merge({ per_page: 100 }),
-			(res) => res.data
-		);
+			const [admin, numberOfRepos] = await Promise.all([checkAdmin, numberOfReposPromise]);
 
-		const [admin, numberOfRepos] = await Promise.all([checkAdmin, repositories]);
+			log.info("Number of repos in the org received via GraphQL: " + numberOfRepos);
```

Comment:
Will be useful to have this number in logs. Without it we had to ask customers directly how many repos they have